### PR TITLE
Correct index.md documentation preamble

### PR DESCRIPTION
**Pull request by joninvski**
Add "---" to indicate the preamble section.

Signed-off-by: Joao Trindade <trindade.joao@gmail.com>

### CODE REVIEW

**Comment by ehazlett** [comment]
Should we have all of the drivers as keywords?  I'm not sure what the recommendation is here.

@SvenDowideit thoughts?

**Comment by SvenDowideit** [comment]
@ehazlett those keywords are put into the html header for  SEO, so I kinda think yes.

File: docs/index.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
+---
 page_title: Docker Machine
 page_description: Working with Docker Machine
 page_keywords: docker, machine, virtualbox, digitalocean, amazonec2
```
Comment (ehazlett) [inline docs/index.md]: according to @SvenDowideit we need each of the drivers here.  thanks!

**Comment by joninvski** [comment]
Added the drivers to the keywords. Skipped the "none" driver for obvious reasons.

**Comment by ehazlett** [comment]
LGTM

**Comment by ehazlett** [comment]
thanks!

**Pull request merged**